recipes/r-microseq/meta.yaml

```diff
@@ -0,0 +1,71 @@
+{% set version = '1.2' %}
+
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-microseq
+  version: {{ version|replace("-", "_") }}
+
+source:
+  fn: microseq_{{ version }}.tar.gz
+  url:
+    - https://cran.r-project.org/src/contrib/microseq_{{ version }}.tar.gz
```

Comment:
Can you please checkout latest master and use the latest bioconductor skeleton. We will get a third URL with the new script.